.github/workflows/test.yml

```diff
@@ -426,3 +426,24 @@ jobs:
         run: ./.ci/helm/buildChart.js
         env:
           VERSION: 0.0.0
+
+  validate-licenses:
+    name: Validate OpenSource Licenses
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Dotnet
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.x
+      - name: Install nuget-licenses tool
+        run: dotnet tool install --global nuget-license
+      - name: Show Allowed Licenses
+        run: cat ./.ci/allowedLicenses.json
+      - name: Show Ignored Packages
+        run: cat ./.ci/ignoredPackages.json
+      - name: Restore Solution
+        run: dotnet restore ./Backbone.sln
+      - name: Validate Licenses
+        run: nuget-license -i ./Backbone.sln -a ./.ci/allowedLicenses.json -ignore ./.ci/ignoredPackages.json -o jsonPretty
```

Comment:
I usually prefer the descriptive flags instead of the abbreviations (e.g. `--input` instead of `-i`). This simplifies the interpretation of the command when you don't know the tool.
And I know that there are probably some commands out there where we don't use the descriptive flags yet. 😉